Knollfear/bcda 740 Static Site

knollfear opened the pull request:
<!-- Replace xxx with the JIRA ticket number: -->
### Fixes [BCDA-740](https://jira.cms.gov/browse/BCDA-740)

<!-- describe the problem being solved here -->
BCDA needs a static one pager of a site.  It will ideally be flexible as needs for additional pages grow so it should be built using jekyll or a similar site builder

### Proposed changes:
<!-- List of changes with bullet points here: -->
Added the static site.  Too many proposed changes to list

### Change Details
<!-- add detailed discussion of changes here: -->
Added jekyll site
added build of site to docker containers
added route for site

### Security Implications
<!-- Does the change deal with PII at all? What should reviewers look for in terms of security concerns? -->
no PII is affected, this just displays a static page.

### Acceptance Validation
<!-- were you able to fully test the acceptance criteria on the related ticket? if not, why not? -->
Run app locally, look at site, click links, etc.  
Run axe Dequeue 508 compliance checker

<!-- insert screenshots if applicable (drag images here) -->
See bcda-740 for Jenn's mockup
![image](https://user-images.githubusercontent.com/42681520/52082234-440d6380-256a-11e9-9488-8286a2956169.png)


### Feedback Requested
<!-- what type of feedback you want from your reviewers? -->
Any

dhgreene commented:
Beautiful and descriptive.  Great work!

- Speaking as one who loves Ruby, can we keep the Jekyll dependencies from loading every time we build the container?  
- The links don't render as links on any of my browsers.
- Interesting property of the VoiceOver utility: it reads a "D" all by itself as a "500."  Thus "Medicare Part B, and Part 500 claims data . . ."

embh commented on bcda-site-static/index.md:
```diff
@@ -0,0 +1,73 @@
+---
+layout: home
+title:  "Beneficiary Claims API"
```
"Beneficiary Claims Data API"?

knollfear commented:
> Beautiful and descriptive. Great work!
> 
>     * Speaking as one who loves Ruby, can we keep the Jekyll dependencies from loading every time we build the container?
> 
>     * The links don't render as links on any of my browsers.
> 
>     * Interesting property of the VoiceOver utility: it reads a "D" all by itself as a "500."  Thus "Medicare Part B, and Part 500 claims data . . ."

I had some malformed markdown.  I have fixed that.  I also removed the build of the site from the process and just checked in the site.  Now if you want to change the site, you need to build and commit that.  It will just be a matter of running a local docker container.

dhgreene commented on bcda-site-static/index.md:
```diff
@@ -0,0 +1,71 @@
+layout: home
+title:  "Beneficiary Claims Data API"
+date:   2017-10-30 09:21:12 -0500
+description: Shared Savings Program ACOs can use this FHIR-based API to retrieve bulk claims data related to their assigned beneficiaries. Under construction; feedback invited.
```
Just double-checking we want the construction message here

knollfear replied:
Elissa confirmed these are the approved words.

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- Metadata for Twitter -->
+	<meta name="twitter:title" content="Beneficiary Claims API" />
```
```suggestion
	<meta name="twitter:title" content="Beneficiary Claims Data API" />
```

knollfear replied:
These should all be fixed.  In general we should avoid making changes in _site as they are built from the files in bcda-site-static.  It was fixed in the config.yml, but I failed to rebuild before committing.

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+	<meta name="twitter:site" content="/" />
+	<meta name="twitter:card" content="summary_large_image">
+	<!-- Facebook / OpenGraph Metadata -->
+  <meta property="og:title" content="Beneficiary Claims API" />
```
```suggestion
  <meta property="og:title" content="Beneficiary Claims Data API" />
```

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+	<meta property="og:image" content="http://localhost:4000/assets/img/default-share-image.jpg" />
+	<meta property="og:url" content="/" />
+
+  <title>Beneficiary Claims API</title>
```
```suggestion
  <title>Beneficiary Claims Data API</title>
```

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+            <div class="ds-u-color--white ds-l-sm-cold--12 ds-l-xl-col--10 ds-u-margin-left--auto ds-u-padding-x--4">
+
+
+                <h1 class="ds-display">Beneficiary Claims API</h1>
```
```suggestion
                <h1 class="ds-display">Beneficiary Claims Data API</h1>
```

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+      -->
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
```suggestion
        Beneficiary Claims Data API
```

embh commented on bcda/_site/index.html:
```diff
@@ -0,0 +1,340 @@
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z ...
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
```suggestion
        Beneficiary Claims Data API
```

embh commented on bcda/_site/404.html:
```diff
@@ -0,0 +1,219 @@
+      -->
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
```suggestion
        Beneficiary Claims Data API
```

embh commented on bcda/_site/404.html:
```diff
@@ -0,0 +1,219 @@
+      <svg fill="none" height="35" viewBox="0 0 35 35" width="35" xmlns="http://www.w3.org/2000/svg"><circle cx="17.5" cy="17.5" r="17" stroke="#323a45"/><circle cx="17.5" cy="17.5" fill="#323a45" r="15.1667"/><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z" fill="#f7f7f7"/><path d="m17.5 19-5-5h3.5v-7h3.5v7h3z" fill="#f7f7f7"/><g stroke="#f7f7f7"><path d="m11.5 18.5h-1.5v6h14.5v-6h-1.5v4h-11.5z ...
+      <!-- Logo Text -->
+      <div class="site-title">
+        Beneficiary Care Data API
```
```suggestion
        Beneficiary Claims Data API
```
